manifest.json

```diff
@@ -341,6 +341,15 @@
       "packageType": "Zip",
       "useCaseName": "Hello World Example"
     },
+    {
```

Comment:
Please remove the changes in this file from the PR. We have frozen this file and won't accept any change in this file.